python/mxnet/ndarray/contrib.py

```diff
@@ -460,3 +461,65 @@ def _to_python_scalar(inputs, type_, name):
         return then_func()
     else:
         return else_func()
+
+def isinf(data, ctx=None):
+    """Checks whether a given NDArray has infinity or not
+
+
+    Parameters
+    ----------
+    input : NDArray
+        An N-D NDArray.
+    ctx : Context
+        Device context of output. Default is current context.
+
+    Returns
+    -------
+    output: NDArray
+        The output NDarray with 1 and 0 indicating whether infinite or not.
```

Comment:
This should specify that the output has the same shape as input, and that it will be 1 where the array element is equal to +/- infinity and is 0 in other places.